### PR TITLE
Update #branding

### PR DESCRIPTION
**Pull request by designatednerd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ After installing, once you restart Xcode, there will be a new option under the `
 RIGHT NOW!!!
 
 #Who
-Initially a [VOKAL](http://www.vokalinteractive.com) Hack Days Fall 2014 Project by [Ellen Shapiro](http://github.com/designatednerd). Massively improved by the desktop programming skillz of [Isaac](http://github.com/ilg) [Greenspan](http://github.com/vokal-isaac). 
+Initially a [Vokal](http://www.vokalinteractive.com) Hack Days Fall 2014 Project by [Ellen Shapiro](http://github.com/designatednerd). Massively improved by the desktop programming skillz of [Isaac](http://github.com/ilg) [Greenspan](http://github.com/vokal-isaac).
```
Comment (designatednerd) [inline README.md]: Note that the link has to remain vokalinteractive for now because vokal.io is still just showing the `pageok` thing.

**Comment by designatednerd** [comment]
@vokal/ios-developers Another "code review"?

**Comment by designatednerd** [comment]
(Note: I've already submitted updated urls to Alcatraz).

**Comment by chillpop** [comment]
(same comment here too)

**Comment by designatednerd** [comment]
@chillpop Which comment?

**Comment by designatednerd** [comment]
@chillpop NM - figured it out and updated.

**Comment by chillpop** [comment]
LGTM

**Pull request merged**